core/objects_test.go

```diff
@@ -92,7 +106,7 @@ func TestChallengeSanityCheck(t *testing.T) {
 				AddressesResolved: []net.IP{net.IP{127, 0, 0, 1}},
 				AddressUsed:       net.IP{127, 0, 0, 1},
 			}}
-			test.Assert(t, chall.IsSane(false), "IsSane should be true")
+			test.Assert(t, chall.IsSane(true), "IsSane should be true")
```

Comment:
I agree that per the comment it was meant to be true, and I see in the logic of the code that that's the correct outcome. But I think the question is: why did this pass before? And what changed so that it now gives correct output?